tests/test_dicom/test_dicom_anonymise.py

```diff
@@ -1,7 +1,84 @@
-from pprint import pprint
-from pydicom import datadict
-from pymedphys.dicom import anonymise_dicom_dataset, IDENTIFYING_KEYWORDS
-from pymedphys.dicom import BaselineDicomDictionary
+
+from os import makedirs, replace, remove, rmdir
+from os.path import abspath, basename, dirname, join as pjoin
+
+import pydicom
+from pymedphys.dicom import (
+    anonymise_dataset,
+    anonymise_directory,
+    anonymise_file,
+    BaselineDicomDictionary,
+    IDENTIFYING_KEYWORDS,
+    is_anonymised_dataset,
+    is_anonymised_directory,
+    is_anonymised_file
+)
+
+HERE = dirname(abspath(__file__))
+DATA_DIR = pjoin(dirname(HERE), 'data', 'dicom_anonymise')
+dicom_test_filepath = pjoin(DATA_DIR, "RP.almost_anonymised.dcm")
+
+
+def test_is_anonymised_dataset():
+    ds = pydicom.dcmread(dicom_test_filepath)
+
+    assert not is_anonymised_dataset(ds)
+
+    ds_anon_manual = ds
+    ds_anon_manual.PatientName = "Anonymous"
+
+    ds_anon_func = anonymise_dataset(ds, delete_private_tags=False,
+                                     copy_dataset=True)
+
+    # Patient Name should've been the only non-anonymous value. If not,
+    # test data has been changed
```

Comment:
Let me know what you think. I find when tests like this highly depend on the data they're passed there is a lot of value in having the data be created by the test itself in a transparent way. All able to be read in the same place.

For example, see: https://github.com/pymedphys/pymedphys/blob/master/tests/test_dicom/test_dicom_base.py#L56-L69

Something like:

```python
import uuid
import os

from pymedphys.dicom import (
    DicomBase, is_anonymised_dataset, is_anonymised_directory)

directory_to_use = str(uuid.uuid4())

names_to_use = ['Python^Monte', 'MacDonald^George']

expected_dicom_file = DicomBase.from_dict({
	'Manufacturer': 'PyMedPhys',
	'PatientName': 'Anonymous'
})

filepaths = []
for i, name in enumerate(names_to_use):
	dicom = DicomBase.from_dict({
	    'Manufacturer': 'PyMedPhys',
	    'PatientName': name
	})
	assert not is_anonymised_dataset(dicom.dataset)
	
	filepath = os.path.join(directory_to_use, "{}.dcm".format(i))
    filepaths.append(filepath)

	dicom.to_file(filepath)

    assert not is_anonymised_file(filepath)


assert not is_anonymised_directory(directory_to_use)
anonymise_file(filepaths[0])
assert is_anonymised(filepaths[0])

# Check that one anonymous file doesn't trigger the directory
assert not is_anonymised_directory(directory_to_use)

anonymise_directory(directory_to_use)
assert is_anonymised_directory(directory_to_use)
assert is_anonymised(filepaths[1])

# etc...	
```